protobuf/scalapb/scalapb.proto

```diff
@@ -2,7 +2,10 @@ syntax = "proto2";
 
 package scalapb;
 
+option csharp_namespace = "ScalaPb.Options";
+option go_package = "scalapb.github.io/pirum.com/cda/grpc/trades";
```

Comment:
The package name here seem to contain identifiers specific to your company or project. I suggest `scalapb.github.io/protobuf/scalapb`

Reply:
woops haha